packages/react/test/specs/components/Menu/Menu-test.tsx

```diff
@@ -118,6 +118,38 @@ describe('Menu', () => {
       })
     })
 
+    describe('variables', () => {
+      it('are passed from Menu to MenuItem and correctly merged', () => {
```

Comment:
Let's mention the MenuDivider in the test as well: 'are passed from Menu to MenuItem and MenuDivider and correctly merged'